layouts/index.html

```diff
@@ -143,5 +156,8 @@ <h3 class="usa-font-lead">All our work is guided by user research — structured
     <!--Footer links END -->
 
     <script src="assets/scripts/main.js"></script>
+
```

Comment:
With two scripts, is it time to create a `script.html` partial? The `index.html` file only gets used for the `/` route and needs to be ported over to `layouts/_default/single.html` eventually for any single pages.